src/components/Story.tsx

```diff
@@ -42,6 +56,7 @@ const styles = {
     position: "relative" as "relative",
     overflow: "hidden",
     alignItems: "center",
+    justifyContent: "center",
```

Comment:
Now that you are allowing styling the inner div you can pass this style as a prop? I think that makes more sense instead of updating it for all existing users.

Reply:
```suggestion
```